torchci/components/benchmark/compilers/BenchmarkLogs.tsx

```diff
@@ -2,26 +2,21 @@ import { LOG_PREFIX } from "components/benchmark/common";
 import { LogLinks } from "components/benchmark/compilers/LogLinks";
 import { SUITES } from "components/benchmark/compilers/SuitePicker";
 import { fetcher } from "lib/GeneralUtils";
-import { RocksetParam } from "lib/rockset";
 import useSWR from "swr";
 
 export const INDUCTOR_JOB_NAME_REGEX = new RegExp(
   ".+\\s/\\stest\\s\\(inductor_(.+)_perf_?(.*), ([0-9]+), ([0-9]+), (.+)\\)"
 );
 
 export function BenchmarkLogs({ workflowId }: { workflowId: string }) {
-  const queryCollection = "commons";
   const queryName = "get_workflow_jobs";
 
   // Fetch the job ID to generate the link to its CI logs
-  const queryParams: RocksetParam[] = [
-    {
-      name: "workflowId",
-      type: "int",
-      value: workflowId,
-    },
-  ];
-  const url = `/api/query/${queryCollection}/${queryName}?parameters=${encodeURIComponent(
+  const queryParams: { [key: string]: any } = {
+    jobName: "%test (%",
+    workflowId: workflowId,
```

Comment:
Does str/int matter here?

Reply:
The preview page looks fine, so I think there is no difference

Reply:
![Screenshot 2024-10-24 at 17 36 01](https://github.com/user-attachments/assets/aec552f0-5b02-486b-975e-2125ca71c11c)

This part shows up ok on https://torchci-git-fork-huydhn-ch-migrate-getworkflowjobs-fbopensource.vercel.app/benchmark/compilers

Reply:
I switch the type of `workflowId` to a number anyway as it looks better